src/cf-base.less

```diff
@@ -246,6 +250,19 @@ h6,
     line-height: unit( @base-line-height-px / @font-size );
 }
 
+.subheader {
+    // For longer subheaders that we don't want to be too large or heavier than
+    // the regular weight.
+    @font-size: 18px;
+
+    margin-top: 0;
+    // Spacing in the design manual is based off of the baseline so this margin
+    // number has been eyeballed to be 30px.
+    margin-bottom: unit( 21px / @font-size, em );
```

Comment:
With settings that are being optically aligned ("eyeballed"), what do you think of formatting the units so they are in the format of `unit( (base - offest) / font-size, unit);`? That way it's clearer what the intended output pixel dimensions are and it'll be easier to spot where different pieces of code share the same margin width, etc. Something like:

```less
margin-bottom: unit( (30px - 9px) / @font-size, em );
```